projects/igniteui-angular/src/lib/date-picker/date-picker.component.ts

```diff
@@ -688,6 +698,12 @@ export class IgxDatePickerComponent implements IDatePicker, ControlValueAccessor
     @ContentChildren(IgxInputDirective, { descendants: true })
     private _inputDirectiveUserTemplates: QueryList<IgxInputDirective>;
 
+    @ViewChild(IgxLabelDirective)
+    private _labelDirectiveDefault: IgxLabelDirective;
```

Comment:
IMO `_labelDirective` without `Default` is ok like name. This is the way we named `_inputGroup` and `_inputDirective`